packages/deployer/test/sample-project/contracts/libraries/UpgradeableProxy.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
```

Comment:
This UpgradeableProxy.sol (as well as Proxy.sol) is too "OpenZeppelin-y". We want something less abstract, more to the point, that uses the namespace architecture directly, and that actually has an anti-bricking mechanism, like this: https://github.com/Synthetixio/synthetix-v3/blob/master/packages/deployer/test/sample-project/contracts/modules/UpgradeModule.sol